readthedocs/__init__.py

```diff
@@ -0,0 +1,5 @@
+from __future__ import absolute_import
+
+# This will make sure the app is always imported when
+# Django starts so that shared_task will use this app.
+from .settings.celery import app as celery_app  # noqa
```

Comment:
Should this not be using an "app config" instead of putting this in the root `__init__`? We could put an `apps.py` in the `core` app similar to: https://github.com/rtfd/readthedocs.org/blob/master/readthedocs/gold/apps.py which runs this code.